src/Raven.Server/Utils/ReplicationUtils.cs

```diff
@@ -125,6 +127,19 @@ public static TcpConnectionInfo GetTcpInfo(string url, string databaseName, stri
             }
         }
 
+        public static async Task<DatabaseStatistics> GetDatabaseStatistics(string url, string database, string apiKey)
+        {
+            JsonOperationContext context;
+            using (var requestExecuter = RequestExecutor.CreateForSingleNode(url, database, apiKey))
```

Comment:
In general, we want to avoid this as much as possible. SingleUse short term is a waste for us in most cases, since we'll be using that over a long period of time